src/lib/updateNotifier.js

```diff
@@ -1,8 +1,3 @@
-const updateNotifier = require('update-notifier');
-const pkg = require('../../package.json');
+const config = require('../config');
 
-const notifier = updateNotifier({
-  pkg,
-  updateCheckInterval: 86400000, // 1 day
-});
-notifier.notify();
+const notifier = config('update_check_interval');
```

Comment:
Hey @codisteinborn You have completely replaced the notifier object.
You are supposed to change only `updateCheckInter : config('update_check_interval');` Please fix it and update your pr